### PR TITLE
Bitbucket Client added

### PR DESCRIPTION
**Pull request by luchianenco**
OAuth Bitbucket Client added, tests added

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
         "symfony/dependency-injection": "^2.8|^3.0",
         "symfony/routing": "^2.7|^3.0",
         "symfony/http-foundation": "^2.7|^3.0",
-        "league/oauth2-client": "^1.0"
+        "league/oauth2-client": "^1.0",
+        "stevenmaguire/oauth2-bitbucket": "^0.1.2"
```
Comment (bocharsky-bw) [inline composer.json]: It's specific to the Bitbucket provider, we don't need to depend on it globally. If users want this feature - they will install it manually. See other providers

Comment (luchianenco) [inline composer.json]: ok I will remove it from global require

**Comment by weaverryan** [comment]
Thanks @luchianenco! Well-done :). I've just released 1.6.0 with this.

Cheers!

**Comment by luchianenco** [comment]
welcome ;)

**Pull request merged**